hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/common/SparkReaderContextFactory.java

```diff
@@ -54,11 +55,13 @@
 import scala.Tuple2;
 import scala.collection.JavaConverters;
 
+import static org.apache.hudi.common.model.HoodieFileFormat.PARQUET;
+
 /**
  * Factory that provides the {@link InternalRow} based {@link HoodieReaderContext} for reading data into the spark native format.
  */
 class SparkReaderContextFactory implements ReaderContextFactory<InternalRow> {
-  private final Broadcast<SparkParquetReader> parquetReaderBroadcast;
+  private final Broadcast<SparkFileReader> parquetReaderBroadcast;
```

Comment:
why do we still call the var name as parquetReaderBroadcast? 
shouldn't we call it as fileReaderBroadcast?